leanCoffree/SessionManager.swift

```diff
@@ -24,8 +24,11 @@ struct SessionManager: View {
         UIScrollView.appearance().keyboardDismissMode = .onDrag
     }
     
-    func goHome() {
-        session = SessionDetails(id: "", localStatus: "WELCOME", sessionStatus: "", dispalyName: "")
+    func changeState(_ state: String) {
+        let sessionId = state == "WELCOME"
+            ? ""
+            : session.id
+        session = SessionDetails(id: sessionId, localStatus: state, sessionStatus: "", dispalyName: "")
```

Comment:
When clicking users icon, this would overwrite displayName with empty string, then when switching back to session backlog view, it wouldn't have displayName and functionality broke. Force re-wrote the commit history to exclude this PR and re-wrote functionality.